README.md

```diff
@@ -101,15 +101,19 @@ none
 - **id** - (string, required): Full filename and path to the file
 
 ### Download File by name
-Finds a file by name in the provided directory and uploads (streams) to the attachment storage
+Finds a file by name in the provided directory and either uploads (streams) its content to the attachment storage or emits its in Base64 representation as a message.
+
 #### Configuration Fields
 * **Allow Empty Result** - (dropdown, optional, defaults to `No`): Do not thrown error when no objects were found
 * **Allow ID to be Omitted** - (dropdown, optional, defaults to `No`): Do not thrown error when object id is missing
+* **Emit file content (Base64)** - (checkbox, optional, defaults to `No`): When checked, a file will not be put into the internal storage. The file content in Base64 representation will be emitted as a part of the message. 
+**Be careful:**
+  * Maximum message size supported on the platform is 10MB. This is why files bigger than 10MB will not be processed.
+  * Retrieve sample of a huge file may freeze the UI.
```

Comment:
It should be fine now - [Handle large data samples on UI](https://docs.elastic.io/releases/24/05.html#handle-large-data-samples-on-ui)